src/msg/msg_types.h

```diff
@@ -462,6 +475,10 @@ struct entity_addr_t {
   // Right now, these only deal with sockaddr_storage that have only family and content.
   // Apparently on BSD there is also an ss_len that we need to handle; this requires
   // broader study
+  // Currently ss_len is set on the decode in set_sockaddr() so it does not need to
+  // be encode to go over the line.
+  // But get_sockaddr_storage() does include ss_len, so it could be that line formats
+  // between Linux and BSD are not compatible
```

Comment:
note that with v2 we aren't send a whole sockaddr_storage over the wire.  does the newer encoding make this issue go away?

Reply:
@liewegas 
I noticed that, but for v1 it is still the case.
So perhaps add that restriction to the text. 
Haven't had time to test interoperability for quite some time, so I could very well be that this is still relevant. Even more with all the v2 changes, and this becoming less obvious.